video/image_writer.c

```diff
@@ -135,19 +135,18 @@ static void prepare_avframe(AVFrame *pic, AVCodecContext *avctx,
     pic->format = avctx->pix_fmt;
     pic->width = avctx->width;
     pic->height = avctx->height;
-    avctx->color_range = pic->color_range =
-        pl_levels_to_av(image->params.repr.levels);
+    pl_avframe_set_repr(pic, image->params.repr);
```

Comment:
I think you meant `_color`

Reply:
No, see below. That's guarded behind tag_csp